packages/dataviews/src/normalize-fields.ts

```diff
@@ -64,6 +64,11 @@ export function normalizeFields< Item >(
 			);
 		};
 
+		const supportsBulkEditing =
```

Comment:
Why bulk editing is tied to the field type or even having a custom edit function? In my view, this is business logic: users can't edit fields that are unique. That is unrelated to any field type or what kind of edit function is in place.

I was looking for something like this in the fields package instead. Additionally, if we agree that the only case in which a field cannot be bulk edited is when it needs to be unique, I wonder how can we tie this to validation. For example, when editing the field, the user shouldn't be able to add a value that is in use by another item.

Reply:
The way I think about this is the following: any field is bulk editable regardless of their fieldType/edit function, unless it provides a specific opt-out (it's an unique field).

Reply:
Some fields could be unique for instance or things like that.